projects/plugins/jetpack/extensions/blocks/ai-assistant/extensions/jetpack-contact-form/ui-handler/with-ui-handler-data-provider.tsx

```diff
@@ -85,7 +113,11 @@ const withUiHandlerDataProvider = createHigherOrderComponent( BlockListBlock =>
 						},
 					} }
 				>
-					<AiAssistantPopover />
+					<AiAssistantPopover
+						onDone={ setContent }
+						onSuggestion={ setContent }
+						askQuestionOptions={ postId }
+					/>
```

Comment:
Never ever pass props down that belong that are accessible from the context.

```suggestion
					<AiAssistantPopover />
```

Instead, just pick the by using the hook:

```jsx
useAiContext( {
	onDone: setContent,
	onSuggestion: setContent,
	askQuestionOptions: { postId },
} );
```